index.py

```diff
@@ -20,16 +24,17 @@ def get_checkbox_errors(checkboxes):
     errors = [
         {"group": default_group, "all": c["all"], "checked": c["checked"]}
         for g, c in checkboxes.items()
-        if (g == default_group and c["checked"] != c["all"])
+        if g == default_group and c["checked"] != c["all"]
     ]
     errors += [
         {"group": g, "all": c["all"], "checked": c["checked"]}
         for g, c in checkboxes.items()
-        if g != default_group and c["checked"] == 0
+        if g != default_group and c["checked"] != c["all"]
```

Comment:
should be == 0, doesnt need to check all the boxes in a group (these are treated as radio groups)